app/views/dockers/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<% @title = "Docker Image Upload" %>
+
+<div class="section">
+  <h4>Upload Docker image</h4>
+  <%= form_with url: { action: :uploadDockerImage }, builder: FormBuilderWithDateTimeInput do |f| %>
+    <p>
+      <%= link_to "Click here", tango_status_course_jobs_path %> to view the list of VM pools currently being used.
+    </p>
+    <%= f.file_field :imageFile, name: "imageFile", button_text: "Choose Image" %>
```

Comment:
this form seems to use the local filenane as the remote filename (and thus image name). It may make sense to allow a target image name to be specified.